scripts/upgrade

```diff
@@ -133,17 +139,6 @@ chown -R $app: "$final_path"
 chmod -R 640 "$final_path"
 find "$final_path" -type d -print0 | xargs -0 chmod 750
 
-#=================================================
-# INSTALL WEKAN NPM DEPENDENCIES
-#=================================================
-ynh_print_info "Installing wekan npm dependencies ..."
-
-# Relaunch a npm install
-pushd $final_path/programs/server
-    ynh_use_nodejs
-    npm install
```

Comment:
Eeeh, why remove this block ? Don't we need to re-run npm install after the sources gets updated ?

Reply:
`npm install` download and install all the modules in node_modules.
The module are already in the wekan-*.*.tar.gz app source used for the upgrade, it's why that file is so huge.

- npm install can took a lot of time
- modules provided in the .tar.gz won't be very old or updated
- we would be sure that the modules provided in the tar.gz work for wekan / more recent can maybe break things in wekan